Presentation.Web/app/utility/kendoGridLauncher.ts

```diff
@@ -158,6 +169,28 @@ module Kitos.Utility.KendoGrid {
                                 }
                             }
                         } as any as kendo.ui.GridColumnFilterable;
+                    case KendoGridColumnFiltering.FixedValueRange:
+                        if (this.valueRange === null) {
+                            throw new Error(
+                                "this.valueRange must be defined when using filtering option FixedValueRange");
+                        }
+                        const valueRange = this.valueRange; //capture the reference to use in lambda below
+                        return {
+                            cell: {
+                                template: (args) => {
+                                    args.element.kendoDropDownList({
+                                        dataSource: valueRange.map(value => {
+                                            return { textValue: value, text: value };
+                                        } ),
+                                        dataTextField: "text",
+                                        dataValueField: "textValue",
+                                        valuePrimitive: true
+                                    });
+                                },
+                                showOperators: false,
+                                operator: this.valueRangeMultiSelect ? "contains" : "eq"
```

Comment:
nogle felter har multi select så der skal man give "true" med til valuerange kaldet og så får man contains i stedet for :)